.travis.yml

```diff
@@ -42,7 +96,9 @@ before_install:
   - if [ "$MOODLE_VERSION" = 36 ] || [ "$MOODLE_VERSION" -le 34 ]; then NVMVERSION=8.9; else NVMVERSION=14.0.0; fi
   - nvm install $NVMVERSION;
   - nvm use $NVMVERSION;
-  - if [ "$CI_PLUGIN" = 2 ]; then
+  - if [ "$CI_PLUGIN" = 3 ]; then
+      composer create-project -n --no-dev --prefer-dist moodlehq/moodle-plugin-ci ci ^3;
+    elif [ "$CI_PLUGIN" = 2 ]; then
```

Comment:
Do we need to keep references to CI_PLUGIN=2? Seems like they are all gone. Or am I missing something?

Reply:
No - no need - have removed them and tidied up the CI_PLUGIN logic in the next commit.